plugins/dmxcontrol3.js

```diff
@@ -0,0 +1,50 @@
+const xml2js = require('xml2js');
+
+module.exports.name = 'DMXControl 3';
+module.exports.version = '0.1.0';
+
+module.exports.import = function importDmxControl3(str, filename, resolve, reject) {
+
+  new xml2js.Parser().parseString(str, (parseError, xml) => {
+    if (parseError) {
+      return reject(`Error parsing '${filename}' as XML.\n` + parseError.toString());
+    }
+
+    let out = {
+      manufacturers: {},
+      fixtures: {},
+      warnings: {}
+    };
+
+    try {
+      const device = xml.device;
+      const info = device.information[0];
+      const manName = info.vendor[0];
+      const manKey = manName.toLowerCase().replace(/[^a-z0-9\-]+/g, '-');
+      out.manufacturers[manKey] = {
```

Comment:
True. Then leave it without a check, we can always manually correct mistakes in the pull request.